Content.Client/DeltaV/Abilities/CrawlUnderObjectsSystem.cs

```diff
@@ -0,0 +1,44 @@
+using Content.Shared.DeltaV.Abilities;
+using Content.Shared.Popups;
+using Robust.Client.GameObjects;
+using DrawDepth = Content.Shared.DrawDepth.DrawDepth;
+
+namespace Content.Client.DeltaV.Abilities;
+
+public sealed partial class HideUnderTableAbilitySystem : SharedCrawlUnderObjectsSystem
+{
+    [Dependency] private readonly AppearanceSystem _appearance = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<CrawlUnderObjectsComponent, AppearanceChangeEvent>(OnAppearanceChange);
+    }
+
+    private void OnAppearanceChange(EntityUid uid,
+        CrawlUnderObjectsComponent component,
+        AppearanceChangeEvent args)
+    {
+        if (!TryComp<SpriteComponent>(uid, out var sprite))
+            return;
+
+        _appearance.TryGetData(uid, SneakMode.Enabled, out bool enabled);
+        if (enabled)
+        {
+            if (component.OriginalDrawDepth != null)
+                return;
+
+            component.OriginalDrawDepth = sprite.DrawDepth;
+            sprite.DrawDepth = (int) DrawDepth.SmallMobs;
+        }
+        else
+        {
+            if (component.OriginalDrawDepth == null)
+                return;
+
+            sprite.DrawDepth = (int) component.OriginalDrawDepth;
+            component.OriginalDrawDepth = null;
+        }
+    }
+}
```

Comment:
This entire system and all associated files will need to be deleted because it does infact conflict with EE code. This system *already exists in this codebase* in a significantly more expanded upon form that anyone can use, and isn't limited to Rodentia.